charts/ckan/values.yaml

```diff
@@ -61,11 +62,67 @@ ckan:
           name: ckan
           key: aws_secret_access_key
 
+  service:
+    annotations: {}
+    port: 80
+
+  nginx:
+    image:
+      repository: public.ecr.aws/nginx/nginx
+      pullPolicy: Always
+      tag: stable-perl
```

Comment:
Once you've gotten rid of govuk-request-id, you can switch back to the regular version without Perl.